server/sonar-process-monitor/src/main/java/org/sonar/process/monitor/JavaProcessLauncher.java

```diff
@@ -33,18 +33,19 @@
 import org.sonar.process.ProcessEntryPoint;
 import org.sonar.process.ProcessUtils;
 
-public class JavaProcessLauncher {
+class JavaProcessLauncher implements AutoCloseable {
```

Comment:
indeed no usage, but why not adding the interface ? It sounds to be a good practice.